<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageLight</class>
 <widget class="QWidget" name="PageLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>163</width>
    <height>142</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="layout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>&amp;Type</string>
     </property>
     <property name="buddy">
      <cstring>cbType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelSource">
     <property name="text">
      <string>&amp;Source</string>
     </property>
     <property name="buddy">
      <cstring>cbSource</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbSource"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelValue">
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>sliderValue</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="sliderValue">
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Main &amp;Color</string>
     </property>
     <property name="buddy">
      <cstring>btnColor</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ColorButton" name="btnColor" native="true">
     <property name="text" stdset="0">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QWidget</extends>
   <header>src/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PageLight</receiver>
   <slot>onLightTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
